projects/ktlint.github.io/package.yml

```diff
@@ -1,18 +1,14 @@
 distributable:
-  url: https://github.com/pinterest/ktlint/releases/download/{{version}}/ktlint-{{version}}.zip
+  url: https://github.com/pinterest/ktlint/releases/download/{{version}}/ktlint
 
 versions:
   github: pinterest/ktlint
 
 warnings:
   - vendored
-
-dependencies:
-  openjdk.org: ^20
```

Comment:
given that this is the only jdk we have (and we don't have `||` dependencies), won't this break the package? or does it not require a jdk at all?

Reply:
Thanks for taking a look at this. It does require a jdk to run, but Java is complicated, and if it does specify the dependency, I don’t think the user will be able to use another jdk (installed through brew or some Java version manager).

In any case, it should at least be changed to ‘*’, as they handle all the Java versions in their executable. I’ll change this mr to that tomorrow, and maybe take a look at how to handle the jdk mess afterwards.

Reply:
You'd likely have to pick a different jdk via careful construction of the path.

Reply:
we can handle complicated we just need to know details, eg. is this a runtime or link time dep? Best open a discussion and we can figure out the best path forwards

Reply:
https://github.com/orgs/pkgxdev/discussions/940 Discussion :)